pages/common/sqsc.md

```diff
@@ -0,0 +1,31 @@
+# sqsc
+
+> A command line AWS Simple Queue Service client.
+
+- List all queues:
+
+`sqsc lq [queue-prefix]`
+
+- List all messages in a queue:
+
+`sqsc ls <queue-name>`
```

Comment:
Same here. `<queue-name>` -> `{{queue-name}}`. The same also goes for the other examples further down.